------------------------------------------------------------
revno: 13230
revision-id: squid3@treenet.co.nz-20160212045316-zwx4r9we4gf27rx3
parent: squid3@treenet.co.nz-20151119051038-vwclso11p68sgvxc
fixes bug: http://bugs.squid-cache.org/show_bug.cgi?id=4431
author: Marcos Mello <marcosfrm@gmail.com>
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.4
timestamp: Fri 2016-02-12 17:53:16 +1300
message:
  Bug 4431: C code is not compiled with CFLAGS
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20160212045316-zwx4r9we4gf27rx3
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.4
# testament_sha1: 53c8d70db04dad826815fb86995cda7e99f1d8e2
# timestamp: 2016-02-12 05:50:53 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.4
# base_revision_id: squid3@treenet.co.nz-20151119051038-\
#   vwclso11p68sgvxc
# 
# Begin patch
=== modified file 'acinclude/compiler-flags.m4'
--- acinclude/compiler-flags.m4	2015-01-09 10:30:44 +0000
+++ acinclude/compiler-flags.m4	2016-02-12 04:53:16 +0000
@@ -36,7 +36,7 @@
   AC_CACHE_CHECK([whether compiler accepts $2],[$1],
   [{
     AC_REQUIRE([AC_PROG_CC])
-    SAVED_FLAGS="$CFLAGS"
+    SAVED_CFLAGS="$CFLAGS"
     SAVED_CXXFLAGS="$CXXFLAGS"
     CFLAGS="$CFLAGS $2"
     CXXFLAGS="$CXXFLAGS $2"
@@ -60,7 +60,7 @@
   AC_CACHE_CHECK([whether compiler requires $2],[$1],
   [{
     AC_REQUIRE([AC_PROG_CC])
-    SAVED_FLAGS="$CFLAGS"
+    SAVED_CFLAGS="$CFLAGS"
     SAVED_CXXFLAGS="$CXXFLAGS"
     AC_COMPILE_IFELSE([AC_LANG_PROGRAM($3,$4)],[$1=no],[],[$1=no])
     if test "x$1" != "xno" ; then

